core/db/migration/migrate-db.sh

```diff
@@ -16,8 +16,14 @@ echo "Implementing Migrations"
 
 psql -c "select addcol('public','users', 'last_login', 'timestamp', 'current_timestamp');"
 psql -c "select addcol('public','media', 'fine_amount', 'real', '0.50');"
+psql -c "CREATE TABLE newsbox (
```

Comment:
@stevethomas19  Will this rewrite or write over this table every time we run this?